app.js

```diff
@@ -47,9 +71,12 @@ app.get('/submissions/:id/for-meeting', async function(req, res, next) {
   }
 
   const meeting = await getMeetingForSubmission(submissionId);
-  return res.status(200).send({
-    data: { id: meeting.id, type: 'meetings', attributes: meeting }
-  });
+  if (meeting?.id) {
+    return res.status(200).send({
+      data: { id: meeting.id, type: 'meetings', attributes: meeting }
+    });
+  }
+  return next({ message: 'The submission is not submitted to any meeting, please contact the administrator', status: 404 }); 
```

Comment:
Had an issue locally with no meetings connected to submission. This errorred so opted to properly return an error in that case (where we need to fix data)